salt/_pillar/metalk8s_endpoints.py

```diff
@@ -0,0 +1,56 @@
+"""Store data about bootstrap services ip/port in pillar"""
+
+import logging
+
+
+log = logging.getLogger(__name__)
+
+__virtualenv__ = 'metalk8s_endpoints'
+
+
+def __virtual__():
+    return __virtualenv__
+
+
+def service_endpoints(service):
+    try:
+        endpoint = __salt__['kubernetes.show_endpoint'](
+            name=service,
+            namespace='kube-system',
+            kubeconfig='/etc/kubernetes/admin.conf',
```

Comment:
These should ideally come from the `ext_pillar` configuration, not hard-coded.

Reply:
I'll move namespace and kubeconfig to ext_pillar configuration